doc/reference/configuration/index.rst

```diff
@@ -28,41 +28,112 @@ Command options
 
 .. _index-tarantool_version:
 
-..  option:: -V, --version
+..  option:: -v, --version
```

Comment:
`-v` and `-V` are supported both: https://github.com/tarantool/tarantool/blob/979b4adb885d5e0953fee9be46873bd779704792/src/main.cc#L717-L720